kids/lib/components/thumbnails/posters/poster_large.dart

```diff
@@ -45,10 +49,41 @@ class PosterLarge extends HookWidget {
                               .shimmer(duration: 1000.ms)
                               .callback(delay: 1000.ms, duration: 250.ms, callback: (c) => true),
                         ),
-                        SizedBox(
-                          height: double.infinity,
-                          child: simpleFadeInImage(url: image!),
-                        ),
+                        hasNewEpisodes
+                            ? Container(
+                                color: design.colors.background2,
+                                child: Transform.translate(
+                                  offset: const Offset(0, -20),
```

Comment:
Since you have a tablet now, can you make the "new episodes" also work for the tablet?